### DISCUSSION TITLE
Null characters in echo requests/responses

### DISCUSSION THREAD

**Discussion by zee-doubleu-ee** [ANLAB-KAIST/KENSv3#136] [Project Q&A 1. Application Layer (Socket)]
Since we are using the newline character as command termination character, it's possible that some requests/responses contain null characters. If the echo requests/responses contain null characters, I think they will not be reflected in submitAnswer().

**Comment by Pusnow** [comment]
The requests and responses do not allow null characters.
You don't need to handle null characters in requests and responses.
